modules/ob-creating-a-s2i-build.adoc

```diff
@@ -12,11 +12,14 @@ You can create a `source-to-image` build and push the created image to a custom
 
 * You have installed the {builds-operator} on the {ocp-product-title} cluster.
 * You have created a `ShipwrightBuild` resource.
+* You have installed the `oc` CLI.
+* You have installed the `shp` CLI, if needed.
```

Comment:
```suggestion
* Optional: You have installed the `shp` CLI, if needed.
```

This raised the question as to why the `oc` CLI is not optional. Proably best to state something like, "Depending on your choice of CLI..."

Reply:
oc CLI is not optional because users need it for a detailed verification in the end. Right now, shp CLI does not support relevant commands for verification.
So, leaving it as is.